packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -55,6 +62,11 @@ class Popover extends React.Component {
      */
     isOpen: PropTypes.bool,
 
+    /**
+     * Function call to handle keydown events,
+     */
+    handleKeyDown: PropTypes.func,
```

Comment:
Could maybe call onKeyDown to match other on(thing happens) function calls, but no biggie if not!

Reply:
agree with @ksmort, the general scheme to follow, 
- if prop: on{Event}
- if function in component: handle{Event}